app/models/adjective.rb

```diff
@@ -0,0 +1,4 @@
+class Adjective < ApplicationRecord
+  has_many :descriptions
+  has_many :users, through: :descriptions, source: :describee
```

Comment:
Are you referring to the has_many :users @kheppenstall ? I doubt we ever would use it, to be honest. I think we just tried to implement every relationship we could see on the database. Easy to remove though. What does @wlffann think?